index.html

```diff
@@ -0,0 +1,214 @@
+            </div>
+                <div class="square">
+              <span>
+                <img src="assets/icons/square.svg" alt="github">
+              </span> 
+                </div>
+              </div>
+              <div class="tesxt-aling">
+              <div class="paragraf">
+                <img src="assets/icons/paragraf.svg" alt="paragraf">
+              </div> 
+              </div>
+              <div class="plane"><img src="assets/icons/plane.svg" alt="plane"></div>
+              </idv>
+              <div class="resume-button">
+                <button class="resume">Get my resume</button>
+              </div>
+              </div>
+             <div class="start">
+
+             </div>
+            <article class="write">
+           <span class="final">
+            <img src="assets/icons/holder.svg" alt="holder">
+           </span>
+           <span class="final-place">
+            <img src="assets/icons/icons.svg" alt="holder">
+            <div class="Language">
+              <p>Language</p>
+            </div>
+            <div class="media">
+              <li>JavaScript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </div>
+           </span>
+           </article>
+            <article class="given">
+                <span class="given-to">
+                <img src="assets/icons/holder.svg" alt="holder">
+                </span>
+                  <span class="pro">
+                    <img src="assets/icons/frame.svg" alt="holder">
+                    <div class="Language">
+                      <p>Frameworks</p>
+                    </div>
+                    <div class="frame">
+                      <p class="react-js">React.js</p>
+                      <p class="ruby-on">Ruby on Rails</p>
+                      <p class="rspec">RSpec</p>
+                    </div>
+                    <div class="frame-plus">
+                      <p class="capybara">Capybara</p>
+                      <p class="selenium">Selenium</div>
+                    </div>
+                    </span>
+            </article>
+            <article class="given">
+              <span class="given-to">
+              <img src="assets/icons/holder.svg" alt="holder">
+              </span>
+                <span class="pro">
+                  <img src="assets/icons/electric.svg" alt="holder">
+                  <div class="Language">
+                    <p>Skills</p>
+                  </div>
+                  <div class="frame">
+                    <p class="react-js">Database Management</p>
+                    <p class="ruby-on">Version Controle</p>
+                  </div>
+                  <div class="frame-plus">
+                    <p class="capybara">CLI</p>
+                    <p class="selenium">WEb Development
+                    <p class="api">API Design</p>
+                  </div>
+                  </span>
+          </article>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, each of them in a `li`. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") should also be a list `ul` and each of them a `li`. This is a project requirement and should be followed as stated. This is good because it improves your page accessibility where assistive technology is able to easily understand and assesses the document. Also, blinks users that depend on using keyboard tab keys to navigate the page will be able to move from one object to the other using the tab key.